sphinx/glossary.rst

```diff
@@ -499,8 +501,8 @@ Glossary
 
    directive
       Directives are used by :term:`batch systems <batch system>` to determine
-      what a :term:`jobs <job>` requirements are, e.g. how much memory it
-      requires.
+      what a :term:`job's <job>` requirements are, e.g. how much memory a
```

Comment:
If you prefer this fair enough (there is nothing wrong with it), the sentence just didn't sound quite right to me given it first refers to the requirements of multiple jobs and then uses 'it' to refer to a single job, but it was probably me being over-critical here.

Reply:
> then uses 'it' to refer to a single job

This is purposeful as we are referring to the requirements of a specific job rather than jobs in general.